mlflow/sklearn/__init__.py

```diff
@@ -455,6 +461,271 @@ def load_model(model_uri):
     )
 
 
+def _get_autolog_training_status():
+    return _autolog_training_status
```

Comment:
Nit: Can we define `_autolog_training_status` above this method? Reading the file sequentially, it's a bit odd to have the variable returned here defined at the global level 200 lines below.